packages/editor/src/client/global.less

```diff
@@ -1,4 +1,32 @@
 .highlighted-row,
 .highlighted-row .rs-table-cell-group .rs-table-cell {
-    background-color: #f2faff;
+  background-color: #f2faff;
```

Comment:
i think we should do the highlighted-row from the duplicating page and article the same way.